playbook.yml

```diff
@@ -53,12 +53,10 @@
 
 - name: Provision prometheus
   become: true
-  vars:
```

Comment:
why do you remove this?
what's the default value of it if the user doesn't set it?

Reply:
If I keep the `vars:` section here it breaks the logic that checks if the value is defined to set a default if it's not. We can put it back as it was but this would break the behavior that Carmine suggested